### ISSUE TITLE
Enable Dropout in inference

### ISSUE THREAD

**Issue by tkramer-motion** [aqlaboratory/openfold#446]
Hello,

I am trying to enable dropout during inference of the multimer model. This is enabled here for AlpaFold - https://github.com/bjornwallner/alphafoldv2.2.0/blob/9f76c2adf55403fd80b9079052716857d77a0396/alphafold/model/model.py#L80. Can you provide a pointer on how to enable this for OpenFold?

Thanks,

Trevor

**Comment by eamag** [comment]
Hi Trevor, 
as I understand, OpenFold uses [PyTorch dropout](https://github.com/aqlaboratory/openfold/blob/3c1fd31ac47c8da54d088badd9eba61fe0b3fd26/openfold/model/dropout.py#L44), and at the [PyTorch docs](https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html) you can see that during eval mode it just computes an identity function. So you don't have to enable anything other than eval mode
Dmitrii
